packages/opsian/opsian.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Low overhead profiling"
+maintainer: "Opsian <admin@opsian.com>"
+authors:
+  "Richard Warburton <richard@opsian.com>, Sadiq Jaffer <sadiq@opsian.com>"
+license: "MIT"
+homepage: "https://github.com/Opsian/opsian-ocaml/wiki"
+bug-reports: "https://github.com/Opsian/opsian-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.12.0"}
+]
+flags: [ verbose ]
+# for building native deps
+depexts: [
+  [ "libpthread-stubs0-dev" ] {os-distribution = "ubuntu" }
+  [ "libc6-dev" ] {os-distribution = "ubuntu" } # -ldl -lrt
+  [ "autoconf" ] {os-distribution = "ubuntu" }
+  [ "libtool" ] {os-distribution = "ubuntu" } # libunwind
+  [ "liblzma-dev" ] {os-distribution = "ubuntu" }
+]
+build: [
+  [ "git" "submodule" "update" "--init" "--recursive" ]
```

Comment:
You shouldn't need a submodule init here, since this package is using a release archive.